pkg/providers/instancetype/instancetype.go

```diff
@@ -178,24 +180,29 @@ func (p *DefaultProvider) UpdateInstanceTypes(ctx context.Context) error {
 	// TODO @joinnis: This can be made more efficient by holding a Read lock and only obtaining the Write if not in cache
 	p.muInstanceTypesInfo.Lock()
 	defer p.muInstanceTypesInfo.Unlock()
-	var instanceTypes []*ec2.InstanceTypeInfo
 
-	if err := p.ec2api.DescribeInstanceTypesPagesWithContext(ctx, &ec2.DescribeInstanceTypesInput{
-		Filters: []*ec2.Filter{
+	var instanceTypes []*ec2types.InstanceTypeInfo
+
+	paginator := ec2.NewDescribeInstanceTypesPaginator(p.ec2api, &ec2.DescribeInstanceTypesInput{
+		Filters: []ec2types.Filter{
 			{
-				Name:   lo.ToPtr("supported-virtualization-type"),
-				Values: lo.ToSlicePtr([]string{"hvm"}),
+				Name:   aws.String("supported-virtualization-type"),
```

Comment:
Can we use `lo.FromPtr` and `lo.ToPtr` everywhere instead of the aws pointer convertor?

Reply:
Do you want me to do that for every one? There are a good amount in the code outside of this PR.